subt/ros/robot/launch/x4.launch

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0"?>
 <launch>
-  <node name="$(arg robot_name)_proxy" pkg="proxy" type="ros_proxy" args="$(arg robot_name)">
-    <remap from="$(arg robot_name)/front_rgbd/image_raw/compressed" to="$(arg robot_name)/front/image_raw/compressed"/>
```

Comment:
why is this line removed?

Reply:
why not?

Reply:
In the commit message I wrote:
> neither ros_proxy_node no the X4 references the topic
> "$(arg robot_name)/front_rgbd/image_raw/compressed"

Reply:
this->subImage = n.subscribe(this->name + "/front/image_raw/compressed", 1000, imageCallback);

Reply:
OK, as discussed on call "/front_rgbd/image_raw/compressed" does not exist for X4

Reply:
Yes, that is what I wrote in the commit message.